ftdetect/gdscript.vim

```diff
@@ -0,0 +1 @@
+autocmd BufNewFile,BufRead *.gd set ft=gdscript
```

Comment:
I'm wondering whether we should do this for the other filetypes that aren't supported by nvim too?

Reply:
I think so.

Reply:
fish, svelte, maybe julia?